File: backend/src/main/java/me/performancereservation/domain/ticket/TicketRepository.java
package me.performancereservation.domain.ticket;

import org.springframework.data.jpa.repository.JpaRepository;

import java.util.List;

public interface TicketRepository extends JpaRepository<Ticket, Long> {
    List<Ticket> findAllByReservationId(Long reservationId);

    List<Ticket> findAllByPerformanceId(Long performanceId);
}

File: backend/src/main/kotlin/me/performancereservation/domain/ticket/TicketRepository.kt
package me.performancereservation.domain.ticket

import org.springframework.data.jpa.repository.JpaRepository

interface TicketRepository : JpaRepository<Ticket, Long> {
    fun findAllByReservationId(reservationId: Long): List<Ticket>

    fun findAllByPerformanceId(performanceId: Long): List<Ticket>
}
